stream-chat-swift-docs/docs/guides/working-with-user.md

```diff
@@ -1,3 +1,136 @@
 ---
 title: Working with User
----
\ No newline at end of file
+---
+
+The User object is accessed through a `Controller`. 
+There is a special type of user object which is `CurrentUser`, representing the currently signed-in user. 
```

Comment:
Would be cool to showcase how some of `ChatUser` properties map to views in DemoApp. For instance name, imageURL, isOnline

Reply:
also, needs explaining "why" we need this special entity